index.js

```diff
@@ -0,0 +1,19 @@
+const Async = require('crocks/Async')
+const axios = require('axios')
+
+function asyncios (params) {
+  const cancelTokenSource = axios.CancelToken.source()
+
+  const opts = Object.assign(
+    {},
+    { cancelToken: cancelTokenSource.token },
+    params
+  )
+
+  return new Async((reject, resolve) => {
```

Comment:
Should we use `function (reject, resolve) {` here since we're using `function` on line 4?

Reply:
In my own world, i use `function` for auto-hosting. Since this is a callback, that won't help much.